### PR TITLE
update v5.2 simulation schema file

### PR DESCRIPTION
**Pull request by tommysitu**
Update simulation schema file to reflect changes in v5.2: 
- matching chaining
- global literals and variables for templating

### CODE REVIEW

**Comment by tommysitu** [comment]
@kapishmalik can I have a second pair of eyes on this one? Has it got all your changes?

**Comment by kapishmalik** [comment]
> @kapishmalik can I have a second pair of eyes on this one? Has it got all your changes?

Sure @tommysitu I ll check and get back.

File: core/handlers/v2/schema.json [hunk 1]
```diff
@@ -43,7 +43,24 @@
 				"matcher": {
 					"type": "string"
 				},
-				"value": {}
+				"value": {},
+				"config": {
+					"properties": {
+						"ignoreUnknown": {
```
Comment (kapishmalik) [inline core/handlers/v2/schema.json:49]: We need to change documentation as well in this file - docs/pages/reference/hoverfly/request_matchers.rst

**Comment by kapishmalik** [comment]
@tommysitu  thanks for the schema version upgrade. Added one small review comment. Rest looks fine to me.

**Pull request merged**